### ISSUE TITLE
autolab should validate the embedded quiz file

### ISSUE THREAD

**Issue by cg2v** [autolab/Autolab#892]
Instructors are trying to upload non-conforming files as embedded quiz data (e.g. pdfs) which triggers a DB exception so they get no feedback. The controller or model should do some basic validation and display a message for uploads that are not appropriate.

Maybe there should also be a checkbox: "I have read and understood the embedded quiz documentation" to enable the upload button.

**Comment by TheodorJ** [comment]
Agreed. I think adding the validation would be the best way to fix this, since the checkbox probably won't get people to read the documentation (like how nobody reads Terms and Conditions before agreeing). Leaving this issue open.
